steps/bitrise-step-saucelabs-espresso-app-automate/0.1.0/step.yml

```diff
@@ -0,0 +1,77 @@
+title: SauceLabs App Automate
+summary: |
+  Upload an app apk and an Espresso test apk to SauceLabs and run tests with App Automate.
+description: |
+  This step uploads a provided app apk and  test apk to SauceLabs.  It then will execute the Espresso Tests for the provided apks in SauceLabs app automate.
+website: https://github.com/moatazeldebsy/bitrise-step-browserstack-espresso-app-automate
+source_code_url: https://github.com/moatazeldebsy/bitrise-step-browserstack-espresso-app-automate
+support_url: https://github.com/moatazeldebsy/bitrise-step-browserstack-espresso-app-automate/issues
+published_at: 2019-10-21T14:32:19.909289+02:00
+source:
+  git: https://github.com/moatazeldebsy/bitrise-step-saucelabs-espresso-app-automate.git
+  commit: 02d031d39bdbfa2be18ed52d327c4d73da29d1ff
+host_os_tags:
+- ubuntu-16.04
+project_type_tags:
+- android
+- iOS
```

Comment:
The CI is failing because of the `iOS` project_type_tag. It should be `ios`
![Screenshot 2019-11-04 at 16 15 53](https://user-images.githubusercontent.com/37296013/68199207-de00b180-ffbd-11e9-9908-6609689689b7.png)

Here is the list:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention